src/libopensc/card-openpgp.c

```diff
@@ -2516,19 +2520,46 @@ pgp_update_new_algo_attr(sc_card_t *card, sc_cardctl_openpgp_keygen_info_t *key_
 
 	if (priv->ext_caps & EXT_CAP_ALG_ATTR_CHANGEABLE) {
 		/* ECDSA and ECDH */
+		/* TODO -DEE could map new OIDs to old OID needed for card here */
```

Comment:
please integrate this comment to the one a couple of lines below to give it more context

Reply:
Removed the comments as `ec_curves_alt` below is used to convert RFC 8410 OID to olde OIDs that are written to the card.